backend/curation/api/curation-api.yml

```
@@ -1301,6 +1325,12 @@ components:
       items:
         $ref: "#/components/schemas/ontology_element"
       type: array
+    dataset_revision_of:
+      description: |
+        If this Dataset is a Revision of another Dataset, then this field contains the Dataset ID of the Dataset that
```

```suggestion
        If this Dataset is a Revision of another Dataset, then this field contains the Dataset id of the Dataset that
```
Convention in this spec is lowercase for the time being 🙂